packages/storybook/stories/AdhocCalling.stories.mdx

```diff
@@ -12,37 +13,44 @@ import AdhocCallingSnippetText from '!!raw-loader!./snippets/AdhocCalling.snippe
 
 # Adhoc calling
 
-<SingleLineBetaBanner version={'1.7.0-beta.1'} />
-
 ## Overview
 
-A call can be started with a Teams user or a call queue by using their id. The id of the Teams user or call queue is
-added to the `CallParticipantsLocator` which is used as the locator in the
-[AzureCommunicationCallAdapterArgs](./?path=/docs/composite-adapters--page#azurecommunicationcalladapterargs)
-when creating your [CallAdapter](path=/docs/composite-adapters--page#call-adapter).
-In the current beta version, only one Teams user can be called. A Teams call queue can also be called. For more
-information on call queues, refer to this
-[Learn documentation](https://learn.microsoft.com/en-us/microsoftteams/create-a-phone-system-call-queue)
+A call can be started with a Teams user or a call queue by using their id. You can call a teams user or a Teams voice application
+like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or, multiple teams users. You cannot call multiple
```

Comment:
```suggestion
like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application or, multiple teams users. You cannot call multiple
```

Reply:
Grammar police time. I think it should be '...like a call queue or auto attendant. These calls it is important to note can only be one Teams voice application, or multiple teams users. You cannot call multiple...'